src/main/java/ch/astorm/jchess/util/UnicodePositionRenderer.java

```diff
@@ -0,0 +1,71 @@
+package ch.astorm.jchess.util;
+
+import ch.astorm.jchess.core.Color;
+import ch.astorm.jchess.core.Moveable;
+import ch.astorm.jchess.core.Position;
+import ch.astorm.jchess.core.entities.*;
+
+import java.io.PrintStream;
+
+public class UnicodePositionRenderer extends AbstractTextPositionRenderer {
+    private final  String lineSeparator = getLineSeparator();
+    private final String emptyDarkCell;
+    private final String emptyLightCell;
+
+    /**
+     * Creates a renderer to the specified {@code out} stream.
+     */
+    private UnicodePositionRenderer(PrintStream out, String emptyDarkCell, String emptyLightCell) {
+        super(out);
+        this.emptyDarkCell = emptyDarkCell;
+        this.emptyLightCell = emptyLightCell;
+    }
+
+    public static void render(PrintStream out, Position position) {
+        new UnicodePositionRenderer(out, "▧", " ").render(position);
+
+    }
+    public static void render(PrintStream out, Position position, String emptyDarkCell, String emptyLightCell) {
+        new UnicodePositionRenderer(out, emptyDarkCell, emptyLightCell).render(position);
+    }
+
+    @Override
+    public CharSequence renderToString(Position position) {
+        StringBuilder sb = new StringBuilder(182);
+        for (var i=7; i>=0; i--) {
```

Comment:
use the number of rows/columns of the board: `position.getBoard().getRowCount()`.